compilation_scripts/cross_compile.sh

```diff
@@ -4,15 +4,20 @@
 ## accepts as argument the path to the toolchain file to be used
 ## if no argument is provided, it looks for a default toolchain in the current directory.
 
+source /root/.bashrc
 
 if [ -z "$1" ]; then
     TOOLCHAIN_PATH=`pwd`/toolchainfile.cmake
+    echo "Using toolchain ${TOOLCHAIN_PATH}"
 else
     TOOLCHAIN_PATH=$1
 fi
 
+# this is needed to avoid https://stackoverflow.com/questions/72978485
+git config --global --add safe.directory '*'
+
 # clear out everything first
-rm -rf build install log
+#rm -rf build install log
```

Comment:
this is temporarily commented out, but I want  to add a command line option to enable/disable it